game/lib/server_list.py

```diff
@@ -0,0 +1,50 @@
+import pygame as py
+import gamestates
+
+
+
+class ServerList:
+    def __init__(self, game, connector):
+        self.__menu_title = "Servers"
+        self.__connector = connector
+        self.__game = game
+        self.__server_list = None
+        self.__active_server = 0
+
+    def loop(self):
+        if self.__server_list is None:
+            self.__server_list = self.__connector.get_servers()
+
+        for event in py.event.get():
```

Comment:
Zamiast zwracać ciągle kolejny stan zmienić stan w obiekcie game tylko kiedy jest to konieczne